src/makeRobot.js

```diff
@@ -38,7 +38,69 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const object = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0, y: 0,
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(step = 1) {
+      let actualStep = step;
+
+      if (step < 0) {
+        actualStep = 0;
+      }
+      this.coords.y += actualStep;
```

Comment:
let's simplify the code in all cases
```suggestion
      if (step >= 0) {
        this.coords.y += actualStep;
      }
```